src/components/DetailPage/DetailPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import MeredithGrey from '../../assets/doctors/MeredithGrey.jpeg';
+// import PropTypes from 'prop-types';
+
+const DetailPage = () => (
+  <>
+    <main className="flex h-5/6 justify-center gap-x-20 pt-20">
+      <div>
+        <img src={MeredithGrey} alt="Dr. Grey" className="rounded-3xl border-4 border-zinc-600" />
+      </div>
+      <aside className="w-3/12 flex flex-col gap-y-5">
+        <div>
+          <p className="text-3xl font-bold text-right">Dr. Meredith Grey</p>
+          <p className="text-lg text-right">General Surgeon</p>
+        </div>
+        <ul>
+          <li className="flex justify-between bg-gray-300 p-2">
+            <p>Consultation Fee:</p>
+            <p>$50</p>
+          </li>
+          <li className="flex justify-between bg-neutral-100 p-2">
+            <p>Experience:</p>
+            <p>20 years</p>
+          </li>
+          <li className="flex justify-between bg-gray-300 p-2">
+            <p>Age:</p>
+            <p>45 years</p>
+          </li>
+        </ul>
+        <p>
+          Meredith Grey is the head of general surgery and former Director of the Residency
+          Program at Grey Sloan Memorial Hospital. She is the daughter of the now-deceased Ellis
+          Grey, a famed general surgeon, and the now-deceased Thatcher Grey.
+        </p>
+        <div className="flex justify-between">
+          <button type="button" className="bg-primary hover:bg-lime-400 p-4 rounded-full text-slate-50">Appointment</button>
+          <button type="button" className="bg-red-600 hover:bg-red-400 p-4 rounded-full text-slate-50">Delete</button>
+        </div>
+      </aside>
+    </main>
+    <footer>
+      <button type="button" className="bg-primary hover:bg-lime-400 font-bold text-3xl text-slate-50 pl-10 p-2  pr-3 rounded-r-full">&#60;</button>
```

Comment:
- [ ] I would highly recommend using icon instead of HTML uni code for Icons because styling it would be a bit easier for as you could see now it's not properly centered (doesn't look like so) and styling it is a bit pain 😂 I could see why you left it.
I would recommend [heroIcons](https://heroicons.com/) since it gives svgs styling them is as simple as just adding another className like so 
`<svg
          xmlns="http://www.w3.org/2000/svg"
          className="h-8 w-8"
          fill="none"
          viewBox="0 0 24 24"
          stroke="currentColor"
          strokeWidth={2}
        >
          <path strokeLinecap="round" strokeLinejoin="round" d="M15 19l-7-7 7-7" />
        </svg>`
        
With Hero Icons | HTML uni code
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/161296781-19a1472f-cdfb-4d0e-ab28-b7cbcfa3603f.png) | ![image](https://user-images.githubusercontent.com/87689547/161296869-ff1f04f8-e272-4826-ade1-8d62d6cd6658.png)

Reply:
Thanks for your feedback, the change is already done! Looks much better with the svc icon 💯